Rectangle/WindowManager.swift

```diff
@@ -122,19 +142,7 @@ class WindowManager {
 
         let resultingRect = frontmostWindowElement.rectOfElement()
         
-        var newCount = 1
-        if lastRectangleAction?.action == calcResult.resultingAction,
-            let currentCount = lastRectangleAction?.count {
-            newCount = currentCount + 1
-            newCount %= 3
```

Comment:
This modulo has been removed, because it duplicates the logic in [RepeatedExecutionsCalculation](https://github.com/rxhanson/Rectangle/blob/c0949bbd4e19c417d351c4d053fb01779c83862e/Rectangle/WindowCalculation/RepeatedExecutionsCalculation.swift#L31), and it could be problematic in future if repeated execution with another modulo is needed.

Reply:
Right, one of those things that I would think about but always forget to change.